lib/deps/clone.js

```diff
@@ -38,6 +38,10 @@ module.exports = function clone(object) {
     return object.toISOString();
   }
 
+  if (object instanceof ArrayBuffer) {
+    return object.slice(0);
+  }
+
```

Comment:
If we're doing this fix for the browser, won't we also need it for Node? I.e. Buffers?